NEWS.rst

```diff
@@ -5,29 +5,131 @@ NEWS
 HEAD
 ====
 
+Features
+--------
+
+- Core
+  - New simplified policy language (670_)
+  - Option to choose between a global and per endpoint connection tracking table (659_)
```

Comment:
- Option to choose between a global (default) and per endpoint connection tracking table (659_)